tests/views/test_mau_views.py

```diff
@@ -30,6 +30,8 @@ class TestSiteMauLiveMetricsViewSet(BaseViewTest):
 
     @pytest.fixture(autouse=True)
     def setup(self, db, settings):
+        if organizations_support_sites():
+            settings.FEATURES['FIGURES_IS_MULTISITE'] = True
```

Comment:
Will this clean up after the test is done? I haven't used [`pytest` settings myself](https://pytest-django.readthedocs.io/en/latest/helpers.html#settings) to know.

Reply:
Good question, I'll find out. For our case it doesn't matter because the 'orgs support sites' is global for the environment since the `edx-organizations` package choice determines what the result of `organizations_support_sites` function call will be